src/main/java/com/techconative/restel/utils/Constants.java

```diff
@@ -56,6 +56,13 @@ private Constants() {}
   public static final String TEST_ASSERTION = "assertion";
   public static final String TEST_FUNCTION = "function";
 
+  // Test api wrapper
+  public static final String TEST_API_WRAPPERS = "test_api_wrappers";
+  public static final String TEST_API_WRAPPER_NAME = "test_api_wrapper_name";
+  public static final String TEST_API_WRAPPER_DESCRIPTION = "test_api_wrapper_description";
+  public static final String TEST_API_WRAPPER_PARAMETERS = "test_api_wrapper_parameters";
```

Comment:
Can we get rid of `test_api_` in the prefix? So that the column name stays concise.

Reply:
and `test_api_wrapper_parameters` can be `wrapper_params`, to be in consistent with `suite_params` and `scenario_params.`

Reply:
Should I refactor the constant name accordingly?